File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_emppayrn.cs
// EXPECTED: 446.50
// Reference emit contract for EMPPAYRN (19h * 23.50, OT 0).
using System;
public static class EmppayrnRef {
  public static void Main() {
    int hours = 19;
    double rate = 23.50;
    double ot = hours < 40 ? 0.0 : 0.25;
    double week = hours * rate * (1 + ot);
    Console.WriteLine(week.ToString("0.00"));
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_emppayrn.py
"""Reference emit for EMPPAYRN — 19h * 23.50, OT 0 → 446.50."""
# EXPECTED: 446.50
HOURS = 19
RATE = 23.50
OT = 0.0 if HOURS < 40 else 0.25
WEEK = HOURS * RATE * (1 + OT)
print(f"{WEEK:.2f}")
